packages/input-checkbox/src/Checkbox.story.js

```diff
@@ -35,14 +35,16 @@ const checkboxStories = (moduleTitle, Checkbox) =>
     .add("in a fieldset", () => (
       <fieldset>
         <Checkbox label="Checkbox Label 1" name="foo" />
-        <Checkbox disabled label="Disabled Checkbox" name="foo" />
+        <Checkbox disabled label="Disabled Checkbox" name="foo" />{" "}
         <Checkbox
           checked
           disabled
           label="Checked Disabled Checkbox"
           name="foo"
         />
         <Checkbox label="Checkbox Label 3" name="foo" />
+        <Checkbox label="Checkbox Label 4" checked name="foo" />
+        <Checkbox label="Checkbox" loading />
```

Comment:
How about adding the loading state to the `with state control` story? So we can have a full example.

Something like this:
```js

    .add("with state control", () => {
      const WithState = () => {
        const [checked, setChecked] = useState(false);
        const [loading, setLoading] = useState(false);

        return (
          <Checkbox
            checked={checked}
            loading={loading}
            onChange={e => {
              setLoading(true);
              setChecked(e.target.checked);
              setTimeout(() => {
                setLoading(false);
              }, 1000);
              action("onChange")(e);
            }}
            label="Checkbox Label"
          />
        );
      };

      return <WithState />;
    })
```

Reply:
Implemented, thank you!